scripts/download

```diff
@@ -4,18 +4,13 @@ cd $(dirname $0)/..
 
 . ./scripts/version.sh
 
-ROOT_VERSION=v0.4.1
+ROOT_VERSION=v0.5.0-rc4
 TRAEFIK_VERSION=1.81.0
 CHARTS_DIR=build/static/charts
 
 mkdir -p ${CHARTS_DIR}
 
 curl --compressed -sfL https://github.com/rancher/k3s-root/releases/download/${ROOT_VERSION}/k3s-root-${ARCH}.tar | tar xf -
-ln -sf pigz bin/unpigz
-for target in iptables iptables-save iptables-restore ip6tables ip6tables-save ip6tables-restore; do
-  ln -sf xtables-legacy-multi bin/$target
-done
-mkdir -p bin/aux && rm bin/mount && ln -sf ../busybox bin/aux/mount
```

Comment:
I understand the thrust of this change is concerned with iptables/nftables but the murder of the `unpigz` symlink and the `aux/mount` move are leaving me scratching my head.

Reply:
alas, they are not murdered, but rather "relocated": https://github.com/rancher/k3s-root/blob/master/scripts/package#L39-L48